### PR TITLE
Boilerplate reorg

### PR DESCRIPTION
**Pull request by brok3turtl3**

### CODE REVIEW

**Review by Ericesposito — APPROVED** [review]
Great work all!

**Review by paglenn — APPROVED** [review]
looks great to me! Thanks for renaming the App component extension to .jsx . It makes things easier to read and understand.

File: client/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (joie-zhang) [inline client/src/App.jsx:1]: Yay! Love this fix :)

**Review by sifulsidd — APPROVED** [review]
Boilerplate looks awesome!

File: client/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      },
+
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [{ loader: "file-loader" }],
+      },
+    ],
+  },
+  resolve: {
+    extensions: [".js", ".jsx"],
```
Comment (joie-zhang) [inline client/webpack.config.js:53]: Good addition!

**Review by joie-zhang — APPROVED** [review]
Great work!

**Pull request merged**